src/Microsoft.Azure.Functions.Worker.Extensions.Mcp/Constants.cs

```diff
@@ -0,0 +1,9 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Microsoft.Azure.Functions.Worker.Extensions.Mcp;
+
+internal static class Constants
+{
+    public const string ToolInvocationContextKey = "ToolInvocationContext";
```

Comment:
This is less feedback and more of a question - would something like a `nameof(ToolInvocationContext)` be appropriate for a constant like this? I don't think it actually matters, but curious if something like that would be considered a problem potentially down the line.